autotest.pm

```diff
@@ -145,6 +145,10 @@ sub loadtest {
 
     return unless $test->is_applicable;
     push @testorder, $test;
+
+    # Test schedule may change at runtime. Update test_order.json to notify
+    # the OpenQA server of the change.
+    write_test_order() if defined($process);
```

Comment:
so I assume the impact is (only) that the file is written some more times?

Reply:
Yes, and only if `loadtest()` gets called at test runtime. The tests scheduled before VM start will be written all at once in `runalltests()` as before.

Reply:
I see, thx.